.github/workflows/test-macos.yml

```diff
@@ -52,12 +52,12 @@ jobs:
       # The `npm ci` for this step fails a lot as part of the Test step. Run it
       # now so that we don't have to wait 2 hours for the Build step to pass
       # first before that failure happens. (And if there's something about
-      # `make run-ci -j2` that is causing the failure and the failure doesn't
+      # `make run-ci -j3` that is causing the failure and the failure doesn't
       # happen anymore running this step here first, that's also useful
       # information.)
       - name: tools/doc/node_modules workaround
         run: make tools/doc/node_modules
       - name: Build
-        run: make build-ci -j2 V=1 CONFIG_FLAGS="--error-on-warn"
+        run: make build-ci -j3 V=1 CONFIG_FLAGS="--error-on-warn"
       - name: Test
-        run: make run-ci -j2 V=1 TEST_CI_ARGS="-p actions --measure-flakiness 9"
+        run: make run-ci -j3 V=1 TEST_CI_ARGS="-p actions --measure-flakiness 9"
```

Comment:
@Trott Can we get the core count using bash, and remove the need to update this workflow in the future? Something like `getconf _NPROCESSORS_ONLN`

Reply:
If we're sure it returns the right number of virtual machines, yeah, that would be great. I'm going to let this run as-is so we can confirm that -j3 is faster than -j2. I'll leave a comment about that on main.